sig-contributor-experience/projects.md

```diff
@@ -0,0 +1,73 @@
+# Projects and Goals
+
+This is a list of the projects and goals currently underway with Contributor Experience. Please submit a PR if you are adding your project to this list. To introduce a new project, attend a weekly meeting or drop a note to us on the mailing list - details can be found on our [README](README.md). 
+
+Want to contribute? Take a look at this list and the "future" at the bottom. Thank you to all of our contributors for your hard work!
+
+## SIG Planning
+
+Project | Owner(s)/Lead(s) | Description | Q1, Q2, Later
+---|---|---|---
+[Charter](https://github.com/kubernetes/community/pull/1847) | SIG Leads | Create our first Charter iteration | Q1
+Goals and Projects | SIG Leads | Create projects.md | Q1
+de-SPOF Community Management | @castrojo | eg: YouTube management | Ongoing
+
+## Mentoring 
+Launch a multi-tier strategy test for group mentoring, mentors on demand (meet our contributors), diversity (outreachy), and google summer of code (students) that promote all levels of the [contributor ladder](community-membership.md).
+
+Project | Owner(s)/Lead(s) | Description | Q1, Q2, Later
+---|---|---|---
+[Group Mentoring](mentoring/group-mentoring.md) | @parispittman | Launch a test cohort that takes 10 current members to reviewer in 3 k8s repos/projects. Develop the learning and development workshops. | Q1 and ongoing
+[Meet Our Contributors](mentoring/meet-our-contributors.md) | @parispittman | Monthly web series similar to user office hours that allows anyone to ask new and current contributors quetions about our process, ecosystem, or their stories in open source  | Q1 - ongoing
+[Outreachy](mentoring/README.md) | @parispittman | Document new features, create new conceptual content, create new user paths | Q1
+[Google Summer of Code](mentoring/google-summer-of-code.md) | @nikitha | Kubernetes participation in Google Summer of Code for students | Q1 - ongoing
+["Buddy" Program](https://github.com/kubernetes/community/issues/1803) | @parispittman, @chrisshort | 1 hour 1:1 sessions for new and current contributors to have dedicated time; meet our contributors but personal | Q2
+
+## Contributor Documentation, Workflow, and Automation
+Ensure the contribution process is well documented, discoverable, and consistent across repos to deliver the best contributor experience. Provide sufficient automation that direct write access to repos is no longer required.
```

Comment:
Provide sufficient automation **so** that direct write access to repos is no longer required?